benchmarks/stable_baselines3/benchfile.py

```diff
@@ -9,19 +9,14 @@ class StableBenchmarkPack(Package):
 
     def install(self):
         """Install requirements one by one, as baselines setup is non-standard"""
+        # box2d-py requires swig
+        self.conda_install("-c", "conda-forge", "swig")
```

Comment:
Can this work? https://pypi.org/project/swig/

Reply:
Interesting I'll check